runtime/compiler/env/VMJ9.cpp

```diff
@@ -2210,21 +2210,21 @@ class TR_MarkHotField : public TR_SubclassVisitor
    TR_MarkHotField(TR::Compilation * comp, TR::SymbolReference * symRef)
       : TR_SubclassVisitor(comp), _symRef(symRef) { }
 
-   void mark(J9Class *, bool);
+   void mark(int32_t frequency, J9Class *, bool);
```

Comment:
Not really the owner of this code, but it would make sense to me that frequency argument (in all the methods introduced) is not the first one. As a min, it should go after the class, if not the last one.